src/gourmet/plugins/browse_recipes/browser.py

```diff
@@ -143,8 +143,8 @@ def build_recipe_model (self, path, val):
             if val is None:
                 searches.append({'column':attr,'search':val,'operator':'='})
             else:
-                searches.append({'column':attr,'search':val})
-        for recipe in self.rd.search_recipes(searches):
+                searches.append({'column': attr, 'search': val})
+        for recipe in self.rd.search_recipes(searches, sort_by=[('title', 1)]):
```

Comment:
This is the meat of the bug

Reply:
Looks something like a patch that I was supplied and added a long time ago to the python2 version. Really thought that it had been fixed in the code base, but apparently not...